recipe/meta.yaml

```diff
@@ -32,22 +32,25 @@ requirements:
     - python    # [win]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [not win]
+    - libgfortran  # [not win]
```

Comment:
Ah, do something meant remove it. Sorry that was really ambiguous. So, yes, let's drop this.

We may need to tinker with this stuff later. I had a lot of fun doing this with SciPy. We may need to do a similar exploration here. Fortunately for you I have done this once now. 😄